### ISSUE TITLE
[Feature Request] Splade Table Header click sort use preserve scroll

### ISSUE THREAD

**Issue by tommy66374** [protonemedia/laravel-splade#289]
```
public function index()
{
    return view('users.index', [
        'users' => SpladeTable::for(User::select(User::DEFAULT_SELECT))
            ->withGlobalSearch('Search', ['id', 'name', ......])
            ->preserveScrollSort() // or enabled by default, add a new withoutPreserveScrollSort() method
            ->defaultSort('id')
            ->column(......)
            ->column(......)
            ->paginate(20),
    ]);
}
```

Otherwise, each time click on the sort in Table Header will jump to top page

Just finished my small project in one week. Love this package so much. Thank you for this wonderful package!!!

**Comment by pascalbaljet** [comment]
Thank you! I think it's a good addition!

**Comment by pascalbaljet** [comment]
Fixed in v1.4.15. Make sure to [update both packages](https://splade.dev/docs/upgrading) (JS + PHP).
